src/sass/components/topictakeaway.scss

```diff
@@ -0,0 +1,63 @@
+.topicTakeawayList ul {
+	list-style:none;
+  padding-left: 1em;
+	position: relative;
+	left: -22px;
+}
+
+.topicTakeawayList ul li {
+  &:before {
+    content: '';
+    display: inline-block;
+    height: 1em;
+    width: 1em;
+    background-image: url('/assets/img/governing-content/tick.svg');
+    background-size: contain;
+    background-repeat: no-repeat;
+    margin-right:0.5em;
+  }
+}
+
+.takeawayPrevCard {
+	background-color: #f7f7f7;
+}
+
+.takeawayNextCard {
+	background-color: #f7f7f7;
+	text-align: right;
+}
+
+.takeawayOutput {
+	margin-left: -8px;
+}
+
+.takeawayCardContent {
+	padding-top: 24px;
+	padding-left: 12px;
+	padding-right: 12px;
+}
+
+.takeawayPaginator {
+	margin-top: 20px;
+	margin-bottom: 40px;
+	font-size: 90%;
+}
+
+.takeawayPaginator a, .takeawayArrows {
+	color: #007099;
+}
+
+.takeawayGray {
+	color: #666666;
+	position: relative;
+	top: -14px;
+	font-weight: lighter;
+}
+
+.centerTakeaway {
+	text-align: center;
+}
+
+.takeawayWhattodoLink {
+	padding-top: 20px;
+}
```

Comment:
We use [BEM](http://getbem.com/) naming conventions. These will need to be fixed up as they are very inconsistent with the other class names conventions on the site. Make sure we don't use camel case it should be a combination of `__` and `-` for BEM.

We also have a sass function for spacing `uikit-space()`. For now lets **not use** it as you will replace it when you update the uikit to the latest version however when writing out pixel values such as `14px` and `20px` that will cause problems in the future.

Whenever we use a colour value `#666` make sure we use the correct variable in the `$uikit-colour-Border`. 

There is also inconsistent white space which is killing me 😬 !!!! Lets make sure we use [`editorconfig`](http://editorconfig.org/) and have consistent white space :)